pkgs/development/python-modules/todoist-api-python/default.nix

```diff
@@ -21,7 +21,7 @@ buildPythonPackage rec {
     owner = "Doist";
     repo = "todoist-api-python";
     tag = "v${version}";
-    hash = "sha256-qOb9qAwjQ0MqR+mdNEkt7W8SiBhZ8gRf01TsHDkLPS4=";
+    hash = "sha256-zDbU+lmr8k5E4q+WrPMiKi+OPfVqKhTXk06UQgLMNs4=";
   };
 
   build-system = [ poetry-core ];
```

Comment:
The build system is `hatchling`, see https://github.com/Doist/todoist-api-python/blob/main/pyproject.toml#L51C1-L53C34